Hypnotic-Herons/sprint-3
# Mk category

## Link to Ticket
Closes
 
## Description of Proposed Changes
-Category button added to navbar
-When clicked, Categories of products appear
-Products are listed under appropriate category
-Number of products in category is listed beside category in parenthesis

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout mk_category
code .

```

1. navigate to the root
1. python manage.py makemigrations website
1. python manage.py migrate
1. in DB Browser, add products/categories as needed
1. python manage.py runserver
1. view http://127.0.0.1:8000/
1. register and log in
1. click on Category button in navbar
1. should see categories of products and products listed underneath, count of products is also listed

## Impacted Areas in Application

- Category button in navbar


## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
